langcodes/__init__.py

```diff
@@ -731,102 +707,91 @@ def describe(self, language=DEFAULT_LANGUAGE, min_score: int=90) -> dict:
         return names
 
     @staticmethod
-    def find_name(tagtype: str, name: str, language: {str, 'LanguageData'}):
+    def find_name(tagtype: str, name: str, language: {str, 'Language', None}=None):
```

Comment:
One thing that Unicode CLDR did right is that they made language names globally unambiguous. We don't need to be told that the name "Italienisch" is in German, because no matter what language it's in, it's only going to refer to Italian. So now we get to ignore the `language` parameter *and* skip all the language matching involved.

If we use language names from other sources such as Wiktionary, we can end up in situations where, for example, "Bemba" means `bem` in English and `bmy` in French. But CLDR is given a higher priority than those in the results, so the unambiguous CLDR name comes first.

Ambiguity can only occur for language names CLDR doesn't know, and even then I haven't found a case where specifying the language the name is in would have resolved the ambiguity. The Slovakian name for Slovenian and the Slovenian name for Slovakian had a close call, but they're still one letter different.